hugo.toml

```diff
@@ -440,12 +440,13 @@ languagedirection = "ltr"
 time_format_blog = "02.01.2006"
 # A list of language codes to look for untranslated content, ordered from left to right.
 language_alternatives = ["en"]
+description = "Orkestrasi Kontainer dengan Skala Produksi"
+languageNameLatinScript = "Bahasa Indonesia"
 
 [languages.hi]
 title = "Kubernetes"
 description = "प्रोडक्शन-ग्रेड कंटेनर ऑर्केस्ट्रेशन"
```

Comment:
This "`languages.hi. description`" may require relocation, similar to the adjustments made for parameters in other languages.

Reply:
Thansk for the review, missed it somehow. Will make changes now.

Reply:
I had one more question on this pr, why i can't see logs from this neltfiy hugo build at link https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/6594a8398c43840008a4b062 It just says "You do not have permissions to view logs for this resource". Is it normal i think for old PRs i saw the logs. Maybe something is changed now. I dont know.